LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Eddie Jaoude Community
+Copyright (c) 2020 Eddie Jaoude 
```

Comment:
```suggestion
Copyright (c) 2020 EddieHubCommunity 
```

Reply:
No, 
I think it should be ***` Eddie Jaoude`***, and also there is a previous conversation regarding that.
You can also check LinkFree Licence - [`Licence`](https://github.com/EddieHubCommunity/LinkFree/blob/6d3b1ce272709bafea85b40cd8a797118751c695/LICENSE#L3).

@SaraJaoude will explain it with more clarity.
cc - @eddiejaoude

Reply:
Ohk!